### ISSUE TITLE
Nodes With Running Jobs

### ISSUE THREAD

**Issue by JohnPTobe** [ngageoint/scale-ui#79]
It was useful in the old ui to tell at a glance which nodes are running jobs and which aren't.  Having a count of the total number of jobs running, or a count per job type and being able to sort the nodes would help with troubleshooting whether jobs are being scheduled properly. If there are a bunch of jobs in the queue but half the nodes don't have jobs running, that's a problem.

**Comment by ericsvendsen-mil** [comment]
This is all currently possible in the UI, aside from sorting based on running jobs.

**Comment by JohnPTobe** [comment]
Unless I'm missing something or this is in an update not currently deployed I have to expand nodes on the v7 nodes page to see if there are any jobs running on a node. I'd like to see at a glance which nodes have jobs running or not. Maybe I want a different page?

**Comment by ericsvendsen-mil** [comment]
Correct, you have to expand the nodes to see this information. Your issue seemed to imply that it wasn't there. Since expanding/collapsing nodes is a relatively simple thing to do, I'll have to talk with Chris to sort out whether it's worth the effort to display that information some other way.

**Comment by JohnPTobe** [comment]
So as we discussed, update the banner for each node to include the running jobs and add sorting by hostname and number of running jobs.

**Comment by ericsvendsen-mil** [comment]
Only piece left here is to sort by hostname and running jobs
